### DISCUSSION TITLE
Any method to automate xattr for quarantine?

### DISCUSSION THREAD

**Discussion by rosslevis** [The-Wineskin-Project/WineskinServer#594] [General]
I distribute a Windows program I wrote via a .zip of a .app wineskin package, but some users get confused how to run the command...
xattr  -drs  com.apple.quarantine  /Applications/Appname.app/
Is there any way to automate this?  I know about .pkg installations but I don't have an installer certificate.
Thanks.

**Comment by emendelson** [comment]
Probably not, but you could use a simpler command. My instructions say:

```
Open a terminal, type this string, followed by a space (and don't forget the space):

xattr -rc

and then drag the app into the terminal and press Return.
```

**Comment by Gcenx** [comment]
You could provide a `.command` file and have them double click it so it’ll run `xattr -rc`

**Comment by rosslevis** [comment]
I believe a .command file would need to be given its own executable attribute before it will run, so no advantage.  I guess there isn't a good solution.

**Reply by Gcenx** [reply]
Yeah……., I’m thinking of when I git clone a repo.

They should be able to secondary click and select open.
